docs/spec/v1beta2/events.md

```diff
@@ -1,5 +1,7 @@
 # Events
 
+<!-- menuweight:20 -->
+
```

Comment:
Shouldn't the `v1beta2` docs all have the same menu weight?

Reply:
The menu weight here defines the order of the page in the menu at https://deploy-preview-1427--fluxcd.netlify.app/flux/components/notification/ and the order should always be: controller options (maintained in the website repo), alerts, events, receivers, providers, API reference.